msteams-platform/concepts/deploy-and-publish/appsource/prepare/frequently-failed-cases.md

```diff
@@ -1,246 +1,333 @@
+
+### 4.1 App manifest
+
+* The app manifest must conform to the [latest manifest schema](~/resources/schema/manifest-schema.md).
+* If your app includes a bot or messaging extension, the manifest must be consistent with Bot Framework metadata, including bot name, logo, privacy policy link, and terms of service link.
+* If your app uses Azure Active Directory (Azure AD) for authentication, include the Azure AD ID in the manifest. For more information, see [manifest schema](~/resources/schema/manifest-schema.md#webapplicationinfo).
+
+### 4.2 App icons
+
+* Your app package must include two PNG versions of your app icon: a color icon and an outline icon.
+* Correctly sized and formatted icons must be specified in your app package. The icons must match what's submitted with the store listing metadata. For more information, see the Teams app [icon guidelines](~/concepts/build-and-test/apps-package.md#app-icons).
+
+### 4.3 App description
+
+The app descriptions in your manifest and store must be the same and follow these guidelines:
+
+* Use "Microsoft Teams" on first reference. Subsequent references can be shortened to ”Teams”.
+* Do not abbreviate "Microsoft" (such as "MS" or "MSFT").
+* Avoid unnecessary capitalization and language errors.
+* Follow the [short description guidelines](~/concepts/deploy-and-publish/appsource/prepare/detail-page-checklist.md#short-description).
+* Follow the [long description guidelines](~/concepts/deploy-and-publish/appsource/prepare/detail-page-checklist.md#long-description).
+* Clearly describe any limitations, conditions, or exceptions to the functionality, features, and deliverables described in the listing and related materials before the customer acquires your offer. The capabilities you declare must relate to the core functions and description of your offer.
+
+### 4.4 Localization
+
+Your package must include a localization file that includes language translations that display based on the Teams language settings. The file must conform to the [Teams localization schema](~/concepts/build-and-test/apps-localization.md).
+
+## 5.0 Tabs
+
+If your app includes a tab, following these guidelines can increase the likelihood your app will pass submission.
+
+For information on creating a high-quality experience, see the [tab design guidelines](~/tabs/design/tabs.md).
+
+### 5.1 Setup
+
+* Tab setup must not dead-end the new user. A way forward message should be provided to complete the workflow.
+* Authentication should happen during tab setup and not after.
+* Tabs must have a help tab with instructions on how to use and configure your app.
+
+### 5.2 Views
+
+* The sign-in screen area use must not use large logos or display an entire webpage.
+* The canvas can be simplified by breaking them into multiple tabs to reduce the complexity.
+* Personal app tabs should not have a duplicate header. Remove the logo from the iframe since the personal app framework already displays the app icon and app name.
+
```

Comment:
@heath-hamilton, Tab should not have left stack column that conflicts with Teams side bar
![image](https://user-images.githubusercontent.com/78354231/109539348-36c75800-7ae7-11eb-97f6-be278bc836cb.png)